tests/unit/script/bluetooth_uuid.rs

```diff
@@ -0,0 +1,131 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use script::dom::bindings::codegen::UnionTypes::StringOrUnsignedLong;
+use script::dom::bindings::str::DOMString;
+use script::dom::bluetoothuuid::BluetoothUUID;
+
+const ADEADBEEF_ALIAS: &'static StringOrUnsignedLong = &StringOrUnsignedLong::UnsignedLong(0xADEADBEEF);
+const AEROBIC_HEART_RATE: &'static str = "aerobic_heart_rate_lower_limit";
```

Comment:
This string is only used to create `DOMString`s can't we just create the DOMString here?